client/layout/masterbar/drafts.jsx

```diff
@@ -64,7 +64,7 @@ class MasterbarDrafts extends Component {
 			<div>
 				<QueryPostCounts siteId={ selectedSite.ID } type="post" />
 				{ this.props.draftCount > 0 &&
-					<Button compact borderless className="masterbar__toggle-drafts" onClick={ this.toggleDrafts } ref="drafts">
+					<Button compact borderless className="masterbar__toggle-drafts" onClick={ this.toggleDrafts } ref="drafts" title={ translate( 'Latest drafts' ) }>
```

Comment:
Hi! I've found a possible matching string that has already been translated **37** times:
`translate( 'Latest Drafts' )` *ES Score: 19*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).